bin/live/pycbc_live_supervise_collated_trigger_fits

```diff
@@ -0,0 +1,626 @@
+        daily_files_dir,
+        ifo
+    ):
+    """
+    Find files which match the specified formats
+    """
+    logging.info(
+        "Finding files in %s with format %s in detector %s",
+        daily_files_dir,
+        daily_fname_format,
+        ifo
+    )
+    combined_days = int(combined_control_options['combined-days'])
+    if 'replay-start-time' in combined_control_options:
+        replay_start_time = int(combined_control_options['replay-start-time'])
+        true_start_time = int(combined_control_options['true-start-time'])
+        replay_duration = int(combined_control_options['replay-duration'])
+        rep_start_utc = lal.GPSToUTC(replay_start_time)[0:6]
+
+        dt_replay_start = datetime(
+            year=rep_start_utc[0],
+            month=rep_start_utc[1],
+            day=rep_start_utc[2],
+            hour=rep_start_utc[3],
+            minute=rep_start_utc[4],
+            second=rep_start_utc[5]
+        )
+
+        td = (day_dt - dt_replay_start).total_seconds()
+
+        # Time since the start of this replay
+        time_since_replay = np.remainder(td, replay_duration)
+
+        # Add this on to the original start time to get the current time of
+        # the replay data
+        true_utc = lal.GPSToUTC(true_start_time)[0:6]
+        dt_true_start = datetime(
+            year=true_utc[0],
+            month=true_utc[1],
+            day=true_utc[2],
+            hour=true_utc[3],
+            minute=true_utc[4],
+            second=true_utc[5]
+        )
+        # Original time of the data being replayed right now
+        current_date = dt_true_start + timedelta(seconds=time_since_replay)
+    else:
+        current_date = day_dt
+
+    date_test = current_date + timedelta(days=1)
+
+    daily_files = []
+    missed_files = 0
+    found_files = 0
+    while found_files < combined_days and missed_files < 10:
+        # Loop through the possible file locations and see if the file exists
+        date_test -= timedelta(days=1)
+        date_out = date_test.strftime("%Y_%m_%d")
+        daily_fname = daily_fname_format.format(
+            date=date_out,
+            ifo=ifo,
+        )
+
+        output_dir = os.path.join(
+            daily_files_dir,
+            date_out
+        )
+        daily_full = os.path.join(
+            output_dir,
+            daily_fname
+        )
+        # Check that the file exists:
+        if not os.path.exists(daily_full):
+            missed_files += 1
+            logging.info("File %s does not exist - skipping", daily_full)
+            continue
+        if not len(daily_files):
+            end_date = date_out
+        # This is now the oldest file
+        first_date = date_out
+        # reset the "missed files" counter, and add to the "found files"
+        missed_files = 0
+        found_files += 1
+        daily_files.append(daily_full)
+
+    if found_files == 0:
+        raise RuntimeError("No files found")
+
+    if missed_files == 10:
```

Comment:
If not too difficult, could we rather take this value from the config file?

Reply:
I have done, but I've made it so that if it isnt given, then 10 is used as a default, as I think that is fairly sensible